src/parallel/bbslocal.cpp

```diff
@@ -215,7 +215,6 @@ void BBSLocal::return_args(int userid) {
     Resource::unref(taking_);
     taking_ = (MessageValue*) ((*i).second);
     keepargs_->erase(i);
-    taking_->init_unpack();
```

Comment:
/!\ I'm really disturb by this

Reply:
This isn't obvious to me. Why do you want to make this change?

Reply:
because the code behind have been removed, it was `unpack_ = first`.

I will dig more, but I think this is because you can abort `unqueuing` if the `n-th` parameter is not what you want.

Reply:
I use a vector and a pointer instead ;)